.travis.yml

```diff
@@ -1,69 +1,120 @@
+    os: linux
+    dist: trusty
+    python: 3.6
+    env:
+    - REPO=https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
+    - OS=linux-64
+  - name: xenial python 3.7
+    os: linux
+    dist: xenial
+    python: 3.7
+    env:
+    - REPO=https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
+    - OS=linux-64
+  - name: xenial python 3.6
+    os: linux
+    dist: xenial
+    python: 3.6
+    env:
+    - REPO=https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
+    - OS=linux-64
+  - name: osx python 3.7
+    os: osx
+    language: generic
+    env:
+    - REPO=https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh
+    - TRAVIS_PYTHON_VERSION=3.7
+    - OS=osx-64
+  - name: osx python 3.6
+    os: osx
+    language: generic
+    env:
+    - REPO=https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh
+    - TRAVIS_PYTHON_VERSION=3.6
+    - OS=osx-64
 before_install:
-  - sudo apt-get update
-  - ldd --version
-  - gcc --version
-  - export START=$(pwd)
+- gcc --version
+- export START=$(pwd)
 install:
-- if [[ "$TRAVIS_PYTHON_VERSION" == "3.7-dev" ]]; then export VADD="py37"; else export VADD="py36";  fi
-- wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
-  -O miniconda.sh
+- wget "$REPO" -O miniconda.sh
 - bash miniconda.sh -b -p $HOME/miniconda
 - export PATH="$HOME/miniconda/bin:$PATH"
 - hash -r
 - conda config --set always_yes yes --set changeps1 no
-- conda install conda-build
-- conda update -q conda
-- conda update -n root conda-build
-- conda config --set anaconda_upload no
 - conda config --append channels conda-forge
 - conda config --append channels tofuproject
 - conda info -a
-- if [[ "$TRAVIS_PYTHON_VERSION" == "3.7-dev" ]]; then export THIS_PY_VERSION="3.7";
-  else THIS_PY_VERSION=$TRAVIS_PYTHON_VERSION;
-  fi
-- conda install -q python=$THIS_PY_VERSION conda-build anaconda-client nose
-  nose-timer coverage codecov
+- conda install -q python="$TRAVIS_PYTHON_VERSION" conda-verify nose nose-timer coverage
+  codecov
 - export REV=$(python -c "import _updateversion as up; out=up.updateversion(); print(out)")
 - export VERSION=$(echo $REV | tr - .)
 - echo $REV
-- conda build conda_recipe
-- export PKG_DIR=$HOME/miniconda/conda-bld/linux-64/
-- conda install tofu --use-local
+- pip install -e ".[dev]"
 script:
 - cd $HOME
-- nosetests tofu.tests --nocapture -v --with-id --with-timer --with-coverage
-  --cover-package=tofu
+- nosetests tofu.tests --nocapture -v --with-id --with-timer --with-coverage --cover-package=tofu
 after_success:
 - codecov
 - chmod +x $START/anaconda_upload.sh
 - echo $TRAVIS_TAG
 before_deploy:
+- echo "BEFORE DEPLOY START........"
 - ls $START
 - cd $START
+- conda config --set anaconda_upload no
+- conda install anaconda-client conda-build
+- conda build conda_recipe
+- export PKG_REAL=$(conda build . --output | tail -1)
+- echo "BEFORE DEPLOY END.........."
 deploy:
-  - provider: script
```

Comment:
If this is not done anymore, where is the package uploaded to anaconda.org ?

Reply:
Forgot to add it ! Sorry and thanks for catching it !

Reply:
(corrected in last commit)